.changeset/lazy-dolphins-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@scalar/api-reference": major
```

Comment:
Can we make this a patch instead of a major release :)

Reply:
Sure thing! Changed it to a patch.